### ISSUE TITLE
[HOLD for payment 2023-07-12] [$1000] Web - Send money - The tab with Bank account and Debit card is in the middle Cash page

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#21259]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to staging.new.expensify.com
2. Log in any account
3. Open any chat
4. Click on the + button in the compose box
5. Choose Send money
6. Enter amount
7. Click Pay with Expensify

## Expected Result:
The tab should be at the bottom

## Actual Result:
The tab with Bank account and Debit card is in the middle Cash page

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.30.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![my issue](https://github.com/Expensify/App/assets/78819774/c61f0dbf-d767-43f6-a0fa-0f6422f9eb06)

https://github.com/Expensify/App/assets/78819774/5ae33eea-986a-4fb0-a017-9dd855f4547a

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0164181e7781294f63</li>
        <li>Upwork Job ID: 1673785107250982912</li>
        <li>Last Price Increase: 2023-06-27</li>
    </ul>
</details>

**Comment by Pujan92** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Send money - The popover with Bank account and Debit card is in the middle Cash page

### What is the root cause of that problem?
Seems the `anchorPositionVertical` here assigned the wrong value here.
[150](https://github.com/Expensify/App/blob/9e8a151b908447f3d79261ddb0f62d10c19b5c6e/src/components/KYCWall/BaseKYCWall.js#L72)

### What changes do you think we should make in order to solve the problem?
We should take out the `- 150` from the `anchorPositionVertical` or reduce it to `- 20` which will reduce the distance from the top.
`anchorPositionVertical: domRect.top,` OR `anchorPositionVertical: domRect.top - 20,`

**Comment by alexpensify** [comment]
This is on my test list

**Comment by alexpensify** [comment]
I've requested a test account in betatify and will test soon

**Comment by alexpensify** [comment]
Assigning External, Tom helped me confirm that this is an issue: 

![image](https://github.com/Expensify/App/assets/8107070/caa83c63-d9c8-4240-8dd2-8cdb0f032126)

**Comment by alexpensify** [comment]
Adding the `Design` label too for feedback to confirm this should be above the button -- thank you!

**Comment by alexpensify** [comment]
For reference, when you click the downcaret in the button for the settlement options menu it's here:

![image](https://github.com/Expensify/App/assets/8107070/ce6b906e-b84f-4f16-9734-c3f79c443b26)

**Comment by shawnborton** [comment]
I think the menu should launch just right above the green button, like 8px above it or something.

**Comment by alexpensify** [comment]
Ok, I'll remove your assignment since we are set on the Design feedback.

**Comment by alexpensify** [comment]
@Santhosh-Sellavel - when you get a chance, can you review the proposal that was shared here: 

https://github.com/Expensify/App/issues/21259#issuecomment-1601628639

Thank you!

**Comment by alitoshmatov** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Web - Send money - The tab with Bank account and Debit card is in the middle Cash page
### What is the root cause of that problem?
When https://github.com/Expensify/App/commit/ac1a3efb803bd36b3d738e7994ec7fb0956eee61 was applied expected behavior changed, before it was `anchorPositionTop` and was was changed to `anchorPositionVertical` which resulted in different behavior since default `anchorAlignment` was bottom.

https://github.com/Expensify/App/blob/05f289b239eb98cabaad316bb19a816ec3468941/src/components/PopoverMenu/index.js#L35-L38

To sum up anchor position was in top of element before and after changes it was in bottom of element. 

### What changes do you think we should make in order to solve the problem?
We should rewrite computation of anchor position considiring above matter
https://github.com/Expensify/App/blob/ef52c5da96ceaaca8e023a6bc7459be967dd0d19/src/components/KYCWall/BaseKYCWall.js#L72-L73
```javascript
anchorPositionVertical: domRect.top - 8,
``` 

### What alternative solutions did you explore? (Optional)

**Comment by Santhosh-Sellavel** [comment]
@Pujan92's [proposal](https://github.com/Expensify/App/issues/21259#issuecomment-1601628639) looks good to me!

C+ Reviewed
🎀 👀 🎀

**Comment by Santhosh-Sellavel** [comment]
@alitoshmatov Thanks for the proposal, but the same has been proposed by @Pujan92 already.

**Comment by alexpensify** [comment]
Reassigning another `bug` team member, I'm going OOO until Monday, July 10, and will take it back then if it's not closed by then. 

Required action from the 🐛 team: Check on the PR state and it might be ready for payment late next week.

**Comment by Pujan92** [comment]
@Santhosh-Sellavel @cristipaval  [PR](https://github.com/Expensify/App/pull/21980) is ready for review!

**Comment by dostongulmatov** [comment]
Hello team,

I am interested if my bug report is eligible for bug report payment because I have reported this issue in [15th of June](https://github.com/Expensify/App/issues/21259) but the same issue was reported by internal team last week and I see that I reported it one week before internal team but the bug which I reported [was considered as duplicate here](https://github.com/Expensify/App/issues/21868). Please check it out and let me know, how the guidelines work for this condition, thank you!

**Comment by alexpensify** [comment]
Alright, removing the other 🐛 team member. I'm back online and see that this is our current state: 

Prepare for payment

**Comment by alexpensify** [comment]
@Santhosh-Sellavel / @Pujan92 - to prepare for payment tomorrow, please apply for this job: 
 
 https://www.upwork.com/jobs/~0164181e7781294f63
 
 Thank you!

**Comment by Pujan92** [comment]
Applied @alexpensify , Thanks!

**Comment by Santhosh-Sellavel** [comment]
@alexpensify I will collect via NewDot!

**Comment by alexpensify** [comment]
@Pujan92 - I've prepared the payment in Upwork.

**Comment by Pujan92** [comment]
> @Pujan92 - I've prepared the payment in Upwork.

@alexpensify accepted, Thanks.

**Comment by alexpensify** [comment]
Alright, @Pujan92 has been paid via Upwork!

**Comment by alexpensify** [comment]
@Santhosh-Sellavel - can you please complete the check list and then I can close this GH. Thank you! 

https://github.com/Expensify/App/issues/21259#issuecomment-1622147879

**Comment by dostongulmatov** [comment]
### Could you please somebody answer my question here, https://github.com/Expensify/App/issues/21259#issuecomment-1615818004

**Comment by Santhosh-Sellavel** [comment]
@alexpensify I will complete it over the weekend, and collect my dues. Caught up with other Priority items. Thanks!

**Comment by Santhosh-Sellavel** [comment]
> ### Could you please somebody answer my question here, [#21259 (comment)](https://github.com/Expensify/App/issues/21259#issuecomment-1615818004)

@alexpensify I think they are eligible for reporting bonus, please have a look and issue them thanks!

**Comment by alexpensify** [comment]
Thanks for flagging, I'm reviewing the timeline and will follow up tomorrow regarding the reporting bonus concerns.

**Comment by dostongulmatov** [comment]
@alexpensify  thank you

**Comment by Santhosh-Sellavel** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [x]  [@Santhosh-Sellavel] The PR that introduced the bug has been identified. Link to the PR: https://github.com/Expensify/App/pull/18692
> * [x]  [@Santhosh-Sellavel] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/18692/files#r1264102939
> * [x]  [@Santhosh-Sellavel] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: No updated need, should have been caught during reviewer testing based on the checklist.
> * [x]  [@Santhosh-Sellavel] Determine if we should create a regression test for this bug. No need, applause had caught this, they might cases for this already!
> * [c]  [@Santhosh-Sellavel] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again. N/A
> * [ ]  [@alexpensify / @lschurr] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

cc: @cristipaval Let me know if you differ on any of the above!

**Comment by alexpensify** [comment]
@dostongulmatov - I've started the process in Upwork for the payment since I see you did report this one. It might have gotten lost in the shuffle but please accept. After, I can complete the payment process. Thanks!

**Comment by dostongulmatov** [comment]
@alexpensify  thank you very much accepted the payment

**Comment by alexpensify** [comment]
Awesome, I've completed the payment process via Upwork!

**Comment by Santhosh-Sellavel** [comment]
Requested Payment on ND

**Comment by alexpensify** [comment]
Thank you for that update. I closed the job in Upwork and am going to close this GH. Great work team!

**Comment by anmurali** [comment]
Approved $1500 (with bonus) for Santhosh
